packages/core/src/external/quest/fhir/diagnostic-report.ts

```diff
@@ -0,0 +1,99 @@
+import { uuidv7 } from "@metriport/shared/util/uuid-v7";
+import {
+  CodeableConcept,
+  DiagnosticReport,
+  Identifier,
+  Observation,
+  Patient,
+  ServiceRequest,
+  Specimen,
+} from "@medplum/fhirtypes";
+import { LOINC_URL } from "@metriport/shared/medical";
+import { ResponseDetail } from "../schema/response";
+import { getPatientReference } from "./patient";
+import { getSpecimenReference } from "./specimen";
+import { getDiagnosticReportCategory } from "../../fhir/resources/diagnostic-report";
+import { getServiceRequestReference } from "./service-request";
+import { getQuestDataSourceExtension } from "./shared";
+import { getObservationReference } from "./observation";
+
+export function getDiagnosticReport(
```

Comment:
@keshavsaharia I know we added the resultComments to the observation, and i think that's mostly correct. I also think we should add them to the diagnostic report for utility's sake.